PhysicsTools/NanoAOD/interface/BTV_flavor.h

```diff
@@ -0,0 +1,7 @@
+int jet_flavour(const pat::Jet& jet,
```

Comment:
why having the header file outside of `PhysicsTools/NanoAOD/plugins/BTVMCFlavourTableProducer.cc` ?  it does not seem to be used elsewhere

Reply:
This is indeed only use in `PhysicsTools/NanoAOD/plugins/BTVMCFlavourTableProducer.cc` this is a copy structure from the original PFNano, and this might be also used in the future for the new ParT v2, PNet integration.

Reply:
remove the BTV_flavor.h